### PR TITLE
Feat/render test/import components

### PR DESCRIPTION
**Pull request by jacob-js**

### CODE REVIEW

File: frontend/src/import/components/FileEntry.test.tsx [hunk 1]
```diff
@@ -1,10 +1,18 @@
+// mute the console
+import 'src/_test_utilities/consoleMock';
```
Comment (abayo-luc) [inline frontend/src/import/components/FileEntry.test.tsx:2]: While I am a big fan of mocking an spying on console functions, I have checked in this file and it looks like we are suppressing the console message completely. Shouldn't we just spy on them and let them show something in the console incase there is something.

Comment (ApostolosBenisis) [reply frontend/src/import/components/FileEntry.test.tsx:2]: true, but they also generate a lot of noise during the tests, 
especially on the backend, as the console is used to log for aws.
What do you suggest for the tests?

**Review by rav3n11 — COMMENTED** [review]
There are more files right? To add these console assertions on? Looking good so far.

File: frontend/src/import/components/FileEntry.test.tsx (hunk 1 above)
Comment (abayo-luc) [reply frontend/src/import/components/FileEntry.test.tsx:2]: I suggest we disable  console message case by case (i.e server logging).

File: frontend/src/import/components/ModelDescriptionField.test.tsx [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
+import 'src/_test_utilities/consoleMock';
+
 import {render, screen} from "src/_test_utilities/test-utils";
 import ModelDescriptionField, {DATA_TEST_ID, TEXT} from "./ModelDescriptionField";
 import {getTestString} from "src/_test_utilities/specialCharacters";
 import {isSpecified} from "src/utils/isUnspecified";
 import React from "react";
 import {typeDebouncedInput} from "src/_test_utilities/userEventFakeTimer";
 
+beforeEach(() => {
```
[the remark this answers is not in the document]
Comment (jacob-js) [reply frontend/src/import/components/ModelDescriptionField.test.tsx:11]: Here we are using `beforeEach` for a specific reason, and the reason is: that we want to make sure we clean any possible call to `console.*` that might have been made by some external libraries.

**Pull request closed without merging**